package.json

```diff
@@ -19,6 +19,7 @@
   "publishConfig": {
     "access": "public"
   },
+  "main": "dist/main.js",
```

Comment:
There's already a `main` in there.
Is it correct?

Reply:
Ah, I completely missed this 🤦 
Though, I am not sure we want to build all the CLI when importing `datadog-ci`. Maybe we could rename `index` to `cli` and keep `index` be what is know `main`? What do you think?

Reply:
I think `dist/index.js` was never meant to be imported.
So I think you can remove it altogether, and keep `"main": "dist/main.js"` only.
The `cli` part is covered by `bin` already.

Reply:
Ah misread your `cli` comment.
I'm fine either way.

For the sake of this PR, I'd keep `dist/index.js` and investigate the renaming in a different PR if ever needed.